tests/e2e/homework/pageObject/poManager.ts

```diff
@@ -0,0 +1,24 @@
+import { Page } from '@playwright/test';
+import { Login } from '../components/loginPage';
+import { StartPage } from '../components/startPage';
+import { PassengerDetailsPage } from '../components/passengerDetailsPage';
+import { SignOff } from '../components/signOff';
+import { PaymentPage } from '../components/paymentPage';
+import { BasePage } from '../components/basePage';
+
+export class PageManager extends BasePage {
```

Comment:
Since you use page manager, as a wizard between pages users sees, extending it with base page defies it's purpose. 
poManager won't have any updated functions from basePage, because manager is not a page.